tests/set-labels/both-additionalLabelFields-fieldSpecs/.expected/results.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
 exitCode: 1
 items:
   - image: gcr.io/kpt-fn/set-labels:unstable
-    stderr: '[error]: failed to configure function: `fieldSpecs` has been deprecated, please rename it to `additionalLabelFields`'
+    stderr: 'failed to evaluate function: `additionalLabelFields` has been deprecated'
```

Comment:
Are we going to to deprecate `additionalLabelFields`?

Reply:
Yes. `set-labels` is a general function and the `additionalLabelFields` is more like a temporary workaround for me. 
To support  `additionalLabelFields`, verifying the custom fields do mean "labels" are necessary. Also, it should avoid overriding the core k8s resources that do not mean "labels". From that perspective, I think we either need  OpenApi schema or let users keep their own function for custom labels, and  `additionalLabelFields` seems not to be the proper solution.

Reply:
`additionalLabelFields` is a short-term solution. Using OpenAPI schema is the long term solution, but this solution is still not complete yet.
`additionalLabelFields` is intended to be a lightweight solution until we have the OpenAPI solution ready. If a user only want to change one more label field in their CRD, it'd be too heavy weight to ask them to build a custom function.

> it should avoid overriding the core k8s resources that do not mean "labels"

This may happen if a user configure it wrong.

IMO if we deprecate it now, we actually increase the barrier for adoption. WDYT?

Reply:
Per offline discussion, we need a more general solution for custom paths rather than relying on OpenApi which has a very long timeline to design and publish new features. 
We will drop additionalLabelFields for now.